ocean_lib/data_provider/data_service_provider.py

```diff
@@ -282,23 +292,27 @@ def start_compute_job(
         payload = DataServiceProvider._prepare_compute_payload(
```

Comment:
does this always have a service with type "compute" or a specific structure? Then I'd like to add an assertion towards that, preventing the user from going through to provider with a bad service.

Reply:
please address this too.